release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -659,6 +659,36 @@ The xref:../scalability_and_performance/psap-node-feature-discovery-operator.ado
 [id="ocp-4-8-dev-exp"]
 === Developer experience
 
+[id="ocp-4-8-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-8-insights-operator-restricted-network"]
+==== Insights Advisor recommendations for restricted networks
+
+In {product-title} 4.8, users operating in restricted networks can gather and upload Insights Operator archives to Insights Advisor to diagnose potential issues. Additionally, users can obfuscate sensitive data contained in the Insights Operator archive before upload.
+
+For more information, see xref:../support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc[Using remote health reporting in a restricted network].
+
+[id="ocp-4-8-insights-operator-widget"]
+==== Insights widget improvements
+
+In {product-title} 4.8, the Insights widget now directs the user to more information in {cloud-redhat-com} when no issues are found. 
```

Comment:
I'm not clear on this one. Maybe that's my ignorance. Does this note address: https://bugzilla.redhat.com/show_bug.cgi?id=1949338? 
If so, can we just say The OpenShift Cluster Manager now correctly reports _) issues found_ when Insights finds when Insights finds no issues. Previously, the Cluster Manager gave no information._

Reply:
This refers to the widget that exists inside of the OCP web console not the one in OCM. So this widget now reports 0 and directs people to OCM when no issues are found via a link. 
Technically the application is called "Insights Advisor" in both places now, so I guess we can say: Insights Advisor in the {product-title} web console now correctly reports 0 issues found. Previously, the Insights Advisor gave no information.

Reply:
or something like that

Reply:
Thank you!

Reply:
@mburke5678 Thanks! change is made.